com.github.hakandundar34coding.mini-system-monitor.yml

```diff
@@ -0,0 +1,106 @@
+app-id: io.github.hakandundar34coding.mini-system-monitor
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: mini-system-monitor
+finish-args:
+  - --share=ipc
+  # Required for monitoring data download/upload speed.
+  - --share=network
+  - --socket=x11
+  # Required for monitoring data download/upload speed.
+  - --device=all
+  # Required for saving application configuration file.
+  - --filesystem=xdg-config
```

Comment:
```suggestion
```

Reply:
How settings can be saved if this line is removed?